collections/_explanations/xia2-merging.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Merging in xia2 — the concept of `WAVELENGTH`
+---
+
+# Merging reflection intensities in xia2 — when a `WAVELENGTH` is not a wavelength
+
+Many crystallographers who use DIALS to integrate their rotation data don't actually interact directly with the DIALS commands.
+Instead, they often use [xia2](https://xia2.github.io) to co-ordinate the different components of the DIALS pipeline automatically.
+Here, we explore the concept of `WAVELENGTH` in xia2 and discover that it doesn't necessarily mean what it says.
+In fact, it turns out simply to be a label denoting a group of sweeps that should be merged together.
+We also learn how to manually assign sweeps of images to such groups, when we want to control how reflection intensities should be merged.
+By exploring what `WAVELENGTH` is, we will also learn a little about the key features of xia2's logical model of a diffraction experiment.
```

Comment:
All of this behaviour is accessed _via_ the use of the `xinfo` file: the default command-line usage of 

```
xia2 (options) (data location)
```

will behave entirely as you may expect

Reply:
This is an _explanation_, rather than a _tutorial_, as categorised here: https://documentation.divio.com/

I don't expect users to come to the knowledgebase looking for user documentation.  This is more a conceptual guide for developers.  It doesn't replace the documentation in xia2.github.io, so if you get to the point of reading this, you presumably already know how to run xia2.